src/settings.js

```diff
@@ -640,25 +640,26 @@ var LZ4 = 0;
 
 // Disables generating code to actually catch exceptions. This disabling is on
 // by default as the overhead of exceptions is quite high in size and speed
-// currently (in the future, wasm should improve that). When exceptions are
+// currently (in the future, wasm should improve that).  When exceptions are
```

Comment:
```suggestion
// currently (in the future, wasm should improve that). When exceptions are
```
I know this is a matter of preference, but this paragraph uses single spaces so I think it'd be better to be consistent within the paragraph.

Reply:
OK.